models/tridimensional/docking_validation/dock_variants.py

```diff
@@ -122,12 +134,16 @@ def dock_variants(pam_variants, path_to_scores, path_to_pdbs='', complex_docking
                         help='path to output directory for variant scores')
     parser.add_argument('--pdb_dir', metavar='D', default='', type=str,
                         help='path to root of variant pdb directories (default: "")')
+    parser.add_argument('--set_partners', metavar='S', default="B_ACD", type=str,
+                        help='string for docking partners (default: "B_ACD")')
+    parser.add_argument('--setup_foldtree', metavar='S', type=str,
+                        help='string for foldtree (default: None - use default ft)')
     parser.add_argument('--complex', metavar='B', nargs='?', const=True, default=False,
-                        type=str, help='[switch] select complex docking (default: "False")')
+                        type=bool, help='[switch] select complex docking (default: "False")')
```

Comment:
can you specify an argument from the command line to be boolean? would the idea in [this](https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse) be better?